packages/client/src/pages/statics/NotFoundPage.re

```diff
@@ -1,6 +1,10 @@
 [@react.component]
 let make = () => {
   <SLayout container=`fluid>
-    <Center> {"Sorry, but the page is not found :(" |> Utils.ste} </Center>
+    <Center>
+      <Icon icon="PACMAN" size="2em" />
```

Comment:
Why pacman?

Reply:
I do not know, just like Pacman. I want to give to the application more interconnection with users through messages and dialogues. To make it more interactive.